shared/fs/row/popup.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import {globalStyles, globalMargins, isMobile} from '../../styles'
+import {Box, Text} from '../../common-adapters'
+import {ModalLessPopupMenu} from '../../common-adapters/popup-menu'
+import PathItemIcon from './path-item-icon'
+import PathItemInfo from './path-item-info'
+import filesize from 'filesize'
+
+type Props = {
+  name: string,
+  size: number,
+  type: Types.PathType,
+  lastModifiedTimestamp: number,
+  lastWriter: string,
+  childrenFolders: number,
+  childrenFiles: number,
+  itemStyles: Types.ItemStyles,
+  menuItems: Array<{
+    title: string,
+    onClick: () => void,
+  }>,
+  onHidden: () => void,
+}
+
+const Popup = ({
+  name,
```

Comment:
lets not destructure all the props of a component if the list is this long. just do props: Props